--- HTML-TagCloud-0.38/lib/HTML/TagCloud.pm	2013-04-20 07:04:45.000000000 +0200
+++ /usr/local/lib/perl/HTML/TagCloud.pm	2015-03-17 19:48:12.000000000 +0100
@@ -14,6 +14,7 @@
         categories                => [],
         levels                    => 24,
         distinguish_adjacent_tags => 0,
+        ignore_case               => 0,
         @_
     };
     bless $self, $class;
@@ -67,6 +68,11 @@
 END_OF_TAG
 }
 
+sub compare_mode {
+    my ( $self ) = @_;
+    $self->{ignore_case} ? "\L$a" cmp "\L$b" : $a cmp $b;
+}
+
 sub tags {
     my ( $self, $limit ) = @_;
     my $counts       = $self->{counts};
@@ -94,7 +100,7 @@
         $factor *= ( scalar @tags / $self->{levels} );
     }
     my @tag_items;
-    foreach my $tag ( sort @tags ) {
+    foreach my $tag ( sort compare_mode @tags ) {
         my $tag_item;
         $tag_item->{name}  = $tag;
         $tag_item->{count} = $counts->{$tag};
